### PR TITLE
Building with Active Record - Micro-Reddit

### PR DESCRIPTION
**Pull request by ad9311**
# Building with Active Record Micro-Reddit

In this PR I present Micro-Reddit</br>
This app contains three models: 
- user
- post
- comment

User can create posts and comments. The later two belong to a user.

### CODE REVIEW

**Review by bini-i — CHANGES_REQUESTED** [review]
# Status: Changes Required :yellow_circle:            
### Hello @ad9311  :wave: 

Your project looks very good and meets almost all the requirements:
#### To Highlight: 
- descriptive and well-written README
- good use of git-flow

Kindly work on the following missing requirements and submit for new review:
#### To improve
- Please make sure there is validation that doesn't allow a comment with an invalid body content, currently a comment with an empty body is allowed.
- Your README describes step by step instructions on how to setup up the project locally, but before migrating the DB, I needed to execute `rails db:create`, kindly add all the steps necessary for someone new to your repo.

#### If you have any questions feel free to ask within the PR and ping me on slack @Binyam Hailemeskel

### HAPPY CODING :computer: :blush: 

> #### Reviewed by Binyam Hailemeskel (Code Reviewer)
> GitHub: [@bini-i](https://github.com/bini-i) 
> Twitter: [@binyamshewa](https://twitter.com/binyamshewa)
> LinkedIn: [LinkedIn](https://www.linkedin.com/in/binyam-hailemeskel-728048151/)#

File: app/models/comment.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  validates :commenter, length: { minimum: 6 }
+  validates :body, length: { maximum: 500 }
```
Comment (bini-i) [inline app/models/comment.rb:6]: Please make sure to check for minimum length too, so that empty body is invalidated.

**Comment by ad9311** [comment]
Validations for a comment's body have been added. Now comments' body should be between 30 to 500 characters.</br>
Also updated README to clarify that is necessary to run `rails db:create` before migration.

**Review by george-shammar — APPROVED** [review]
### **STATUS: APPROVED** :heavy_check_mark: :heavy_check_mark: 

@ad9311 , Good job and thanks for implementing the required changes from he last review. :clap: 

**Highlights**

- [ ] Your comment's body is now validated for minimum characters such that an empty comment cannot be submitted.
- [x] You've also updated your README as requested.

[OPTIONAL]

- [x] You may want to lower the minimum character count for the comment's body. What if a commenter has a really short but valid comment (e.g "Great job"). Your app would keep throwing up errors which could frustrate genuine users.  

You may go ahead and merge this branch.
I wish you good luck with your future projects.

Cheers :smile:

**Pull request merged**